src/rt/minfo.d

```diff
@@ -457,10 +457,15 @@ struct ModuleGroup
 
             // add this module to the construction order list
             ctors[ctoridx++] = current;
+            return true;
         }
 
-        immutable(ModuleInfo)*[] doSort(size_t relevantFlags)
+        // `cycle` is set to `true` if deprecated cycle error otherwise set `result`.
+        void doSort(size_t relevantFlags, ref bool cycle, ref immutable(ModuleInfo)*[] result)
```

Comment:
Not sure what you mean, you just would return a bool. But not important right now.